gift_compassion/models/contracts.py

```diff
@@ -77,15 +78,42 @@ def open_gifts(self):
         }
 
     def hold_gifts(self):
-        """ Postpone open gifts. """
+        for contract in self:
+            today = datetime.date.today()
+            first_day = today.replace(day=1)
+            last_day = today.replace(day=calendar.monthrange(today.year,
+                                                             today.month)[1])
+            suspended_gifts = self.env['sponsorship.gift'].search([
+                ('child_id.project_id', '=', contract.project_id.id),
+                ('state', '=', 'Delivered'),
+                ('create_date', '>=', str(first_day)),
+                ('create_date', '<=', str(last_day))
+            ])
+            suspended_gifts.action_suspended()
+
+            related_move = self.env['account.move'].browse(
```

Comment:
You don't need to use the browse method. You can simply use suspended_gifts.mapped('payment_I'd)